hack/cherry_pick_pull.sh

```diff
@@ -123,18 +123,31 @@ for pull in "${PULLS[@]}"; do
 done
 gitamcleanup=false
 
+function make-a-pr() {
+  echo "+++ Now you must propose ${NEWBRANCH} as a pull against ${BRANCH} (<--- NOT MASTER)."
+  echo "    You are constructing a pull against the upstream release branch! To do"
+  echo "    this in the GitHub UI, when you get to the 'Comparing Changes' screen, keep the"
+  echo "    'base fork' as GoogleCloudPlatform/kubernetes, but change the 'base' to e.g. 'release-1.0',"
+  echo "    presumably ${BRANCH}. (This selection is near the top left.)"
```

Comment:
I'm still confused. What will ${BRANCH} evaluate to if you follow the steps I did earlier? Will it say 'release-1.0' or something else?